src/pilet-debug.test.ts

```diff
@@ -1,6 +1,9 @@
-import { cliVersion, runTests, selectedBundler, getFreePort } from './utils';
+import axios from 'axios';
+import { cliVersion, runTests, selectedBundler, getFreePort, ServerRunner } from './utils';
 
 runTests('pilet-debug', ({ test, setup }) => {
+  let server: ServerRunner;
```

Comment:
How is that used? I don't think its used at all? We don't need a custom feed server here, or?